src/Microsoft.AspNet.Mvc.Core/Filters/IFormatFilter.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    ///  A filter which produces a desired content type for the current request. 
+    /// </summary>
+    /// <remarks>A FormatFilter decides what content type to use if a format value is present in the URL. 
```

Comment:
oh I thought you said it could go to remarks.... from summary. That's the change I made. I will remove it altogether.